app/controllers/movies_controller.rb

```diff
@@ -0,0 +1,20 @@
+class MoviesController < ApplicationController
+  def search
+    query = params[:q]
+    page = params.key?(:page) ? params[:page] : 1
+
+    @tmdb_client = TheMovieDbClient.new
```

Comment:
Instead of creating a new instead on each method, you could do it on MoviesController constructor